doc/devel/triage.rst

```diff
@@ -0,0 +1,214 @@
+
+.. _bug_triaging:
+
+Bug triaging and issue curation
+================================
+
+The `issue tracker <https://github.com/matplotlib/matplotlib/issues>`_
+is important to the communication in the project: it is where users to
+make feature requests and report bugs, developers identify major
```

Comment:
I am totally anti the dev/user split, especially since like we don't even have these breakdowns in the issue trackers. Folks with commit rights open tons of bugs and feature requests, discussions about priorities tend to happen amongst whoever is commenting in that issue. suggested instead:
> The `issue tracker <https://github.com/matplotlib/matplotlib/issues>`_ is important to communication in the project because it serves as the centralized location for making feature requests, reporting bugs, identifying major projects to work on, and discussing priorities.

Reply:
I agree with the dev/user split being bad in general.  This particular wording didn't bother me too much as different people in the community do have different responsibilities, but have taken your wording here.